Package.swift

```diff
@@ -25,38 +25,71 @@ let package = Package(
             ),
             exclude: ["001-warnings-and-data-race.patch"]
         ),
-        .target(name: "CSQLite", cSettings: [
-            // Derived from sqlite3 version 3.43.0
-            .define("SQLITE_DQS", to: "0"),
-            .define("SQLITE_ENABLE_API_ARMOR"),
-            .define("SQLITE_ENABLE_COLUMN_METADATA"),
-            .define("SQLITE_ENABLE_DBSTAT_VTAB"),
-            .define("SQLITE_ENABLE_FTS3"),
-            .define("SQLITE_ENABLE_FTS3_PARENTHESIS"),
-            .define("SQLITE_ENABLE_FTS3_TOKENIZER"),
-            .define("SQLITE_ENABLE_FTS4"),
-            .define("SQLITE_ENABLE_FTS5"),
-            .define("SQLITE_ENABLE_MEMORY_MANAGEMENT"),
-            .define("SQLITE_ENABLE_PREUPDATE_HOOK"),
-            .define("SQLITE_ENABLE_RTREE"),
-            .define("SQLITE_ENABLE_SESSION"),
-            .define("SQLITE_ENABLE_STMTVTAB"),
-            .define("SQLITE_ENABLE_UNKNOWN_SQL_FUNCTION"),
-            .define("SQLITE_ENABLE_UNLOCK_NOTIFY"),
-            .define("SQLITE_MAX_VARIABLE_NUMBER", to: "250000"),
-            .define("SQLITE_LIKE_DOESNT_MATCH_BLOBS"),
-            .define("SQLITE_OMIT_DEPRECATED"),
-            .define("SQLITE_OMIT_LOAD_EXTENSION"),
-            .define("SQLITE_OMIT_SHARED_CACHE"),
-            .define("SQLITE_SECURE_DELETE"),
-            .define("SQLITE_THREADSAFE", to: "2"),
-            .define("SQLITE_USE_URI"),
-        ]),
-        .target(name: "SQLiteNIO", dependencies: [
-            .target(name: "CSQLite"),
-            .product(name: "Logging", package: "swift-log"),
-            .product(name: "NIO", package: "swift-nio"),
-        ]),
-        .testTarget(name: "SQLiteNIOTests", dependencies: ["SQLiteNIO"]),
+        .target(
+            name: "CSQLite",
+            cSettings: sqliteCSettings
+        ),
+        .target(
+            name: "SQLiteNIO",
+            dependencies: [
+                .target(name: "CSQLite"),
+                .product(name: "Logging", package: "swift-log"),
+                .product(name: "NIOCore", package: "swift-nio"),
+                .product(name: "NIOPosix", package: "swift-nio"),
+                .product(name: "NIOFoundationCompat", package: "swift-nio"),
+            ],
+            swiftSettings: swiftSettings
+        ),
+        .testTarget(
+            name: "SQLiteNIOTests",
+            dependencies: [
+                .target(name: "SQLiteNIO"),
+            ],
+            swiftSettings: swiftSettings
+        ),
     ]
 )
+
+var swiftSettings: [SwiftSetting] { [
+    .enableUpcomingFeature("ConciseMagicFile"),
+    .enableUpcomingFeature("ForwardTrailingClosures"),
+    .enableUpcomingFeature("DisableOutwardActorInference"),
+    .enableExperimentalFeature("StrictConcurrency=complete"),
```

Comment:
This needs to be in 5.9 right?

Reply:
It is.

Reply:
Right but won't work in the 5.8 manifest

Reply:
Neither will `DisableOutwardActorInference`, apparently. (Which is moot since we don't use any actor annotations anyway.) `ConciseMagicFile` is nice to have though (and is valid in 5.8).